tests/foreman/ui/test_contentview.py

```diff
@@ -4297,3 +4298,15 @@ def test_positive_greedy_dep_solving_with_multiversion_packages(session, module_
             'name = {}'.format(property_name),
             conserve_param_value
         )
+
+
+@tier2
+def test_positive_create_container(session, module_org):
+    """ Create container and register to satellite
+
+    id: 39842da3-94b4-4348-b7d9-94282ae4aaf8
+    """
```

Comment:
your test docstring needs some more fields added.
"Docstring should have at least caseautomation, casecomponent, caseimportance, caselevel, id, requirement, testtype, upstream token(s)"

Reply:
Added setup, caselevel, case importance.  Honestly most of these are already at the top of the file and when we did docstring clean up, it wasn't necessary to repeat some of the case importance or case level if it's already at the top.  It's only required if the author thinks it needs to be changed.